internal/pkg/template/input_test.go

```diff
@@ -143,28 +143,28 @@ func TestTemplateInputsJsonUnmarshal(t *testing.T) {
 
 	var inputs Inputs
 	assert.NoError(t, json.Unmarshal([]byte(inputsJson), &inputs))
-	assert.Len(t, inputs, 3)
-
-	assert.Equal(t, &Input{
-		Id:          "fb.extractor.username",
-		Name:        "Facebook username",
-		Description: "Facebook username description",
-		Type:        "string",
-		Kind:        "input",
-	}, inputs[0])
-	assert.Equal(t, &Input{
-		Id:          "fb.extractor.password",
-		Name:        "Facebook password",
-		Description: "Facebook password description",
-		Kind:        "password",
-	}, inputs[1])
-	assert.Equal(t, &Input{
-		Id:          "fb.extractor.options",
-		Name:        "Facebook options",
-		Description: "Facebook options description",
-		Kind:        "select",
-		Options:     []Option{"1", "2"},
-	}, inputs[2])
+	assert.Equal(t, Inputs{
```

Comment:
Commitol som, co som tym komentarom predtym myslel.
Zbytocne to porovnavat osobitne + len.

Reply:
Aby som ta uz neotravoval; https://github.com/keboola/keboola-as-code/pull/397/commits/12d451331dd76025c03a9f194214f80229aaee02

Reply:
omg 🙈 to mě nějak nenapadlo. 😅